avalanche/training/templates/base.py

```diff
@@ -52,15 +54,13 @@ def __init__(
         ###################################################################
         # State variables. These are updated during the train/eval loops. #
         ###################################################################
-        self.experience: Optional[ClassificationExperience] = None
+        self.experience: Optional[CLExperience] = None
         """ Current experience. """
 
         self.is_training: bool = False
         """ True if the strategy is in training mode. """
 
-        self.current_eval_stream: Optional[
-            Sequence[ClassificationExperience]
-        ] = None
+        self.current_eval_stream: Optional[CLStream] = None
```

Comment:
this is taken from the train and eval arguments, so it should be an `ExpSequence`

Reply:
nice one, I'll fix it right up